common/presenters/move-to-card-component.test.js

```diff
@@ -464,6 +466,66 @@ describe('Presenters', function () {
           })
         })
       })
+
+      context('with identifier number', function () {
+        context('when to_location is a prison', function () {
```

Comment:
I think we're missing a few test scenarios with the current code. For example, when no profile exists or no `to_location` exists. But probably best to wait for clarity on what happens when there is no value and then writing the test cases for those scenarios.